src/OpenTelemetry.Instrumentation.Http/CHANGELOG.md

```diff
@@ -34,6 +34,23 @@
   `http.client.request.duration` metrics on .NET Framework for `HttpWebRequest`.
   ([#4870](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4870))
 
+* Following `HttpClient` metrics will now be enabled by default when targeting
+  `.NET8.0` framework.
+
+  * **Meter** : `System.Net.Http`
+    * `http.client.request.duration`
+    * `http.client.active_requests`
+    * `http.client.open_connections`
+    * `http.client.connection.duration`
+    * `http.client.request.time_in_queue`
+
+  * **Meter** : `System.Net.NameResolution`
+    * `dns.lookups.duration`
```

Comment:
Heads up that this will be renamed to `dns.lookup.duration` in .NET 8.0 GA!

Reply:
Thanks @antonfirsov. We will update the doc accordingly then.